kqueen_ui/blueprints/ui/views.py

```diff
@@ -197,43 +197,72 @@ class UserInvite(KQueenView):
     methods = ['GET', 'POST']
 
     def handle(self):
-        form = UserInviteForm()
+        form_cls = UserInviteForm
+        auth_options = app.config.get('AUTH_OPTIONS', {})
+        if auth_options:
+            auth_choices = []
+            for name, options in auth_options.items():
+                choice = (name, options.get('label', name))
+                auth_choices.append(choice)
+            field_kw = {
+                'auth_method': {
+                    'type': 'select',
+                    'label': 'Authentication Method',
+                    'choices': auth_choices,
+                    'validators': {
+                        'required': True
+                    }
+                }
+            }
+            form_cls.append_fields(field_kw)
+        form = form_cls()
         if form.validate_on_submit():
             organization = 'Organization:{}'.format(session['user']['organization']['id'])
-            password = generate_password()
+            auth_method = 'local'
+            notify = True
```

Comment:
its redundant coz in a few lines below u already set default value

Reply:
Same as below

Reply:
ok